src/System.Data.SqlClient/src/Interop/SNINativeMethodWrapper.Common.cs

```diff
@@ -8,6 +8,9 @@ namespace System.Data.SqlClient
 {
     internal static partial class SNINativeMethodWrapper
     {
+#if NET8_0_OR_GREATER
+        [Obsolete("Use Microsoft.Data.SqlClient package instead.")]
```

Comment:
Can you please move this english string to the Strings.resx file instead and consume it from there to avoid repetition?

Since this is an actual new source code change with a new resource string, I think we might have to add maintenance-packages to the Loc process so that this gets translated, and having the string in the resx might be a requirement.

Reply:
BTW I'd change it this string to:
```
Use the Microsoft.Data.SqlClient package instead.
```

Reply:
Attribute parameters must be constants and afaik can't be localized. I don't think we localize them in runtime either.